packages/shared-ux/code_editor/impl/code_editor.tsx

```diff
@@ -701,23 +700,6 @@ const useFitToContent = ({
   }, [editor, isFitToContent, minLines, maxLines, isFullScreen]);
 };
 
-// https://github.com/elastic/kibana/issues/175684
-// 'react-monaco-editor' has a bug that it always calls the initial onChange callback, so the closure might become stale
-// we work this around by calling the latest onChange from props
-const useBug175684OnChange = (onChange: CodeEditorProps['onChange']) => {
```

Comment:
this also fixes this bug in the react_monaco_editor, so this workaround in the parent component is no longer needed